relative-json-pointer.xml

```diff
@@ -83,35 +83,31 @@
             <t>
                 A Relative JSON Pointer is a Unicode string in UTF-8 encoding (see RFC 8259,
                 <xref target="RFC8259">Section 8</xref>), comprising a non-negative integer,
-                followed by either a '#' (%x23) character or a JSON Pointer
-                (<xref target="RFC6901">RFC 6901</xref>).
+                an optional index adjustment consisting of '+' (%x2B) or '-' (%x2D) followed
+                by a positive integer, followed by either a '#' (%x23) character or
+                a JSON Pointer (<xref target="RFC6901">RFC 6901</xref>).
             </t>
             <t>
-                The separation between the integer prefix and the JSON Pointer will
+                The separation between the integer prefix (with optional adjustment)
+                and the JSON Pointer will
                 always be unambiguous, because a JSON Pointer must be either zero-
                 length or start with a '/' (%x2F).  Similarly, a JSON Pointer will
                 never be ambiguous with the '#'.
             </t>
-            <figure align="center">
-                <preamble>
-                    The ABNF syntax of a Relative JSON Pointer is:
-                </preamble>
-                <artwork>
-<![CDATA[
-   relative-json-pointer =  non-negative-integer [index-manipulation] <json-pointer>
-   relative-json-pointer =/ non-negative-integer "#"
-   index-manipulation    =  ("+" / "-") non-negative-integer
-   non-negative-integer      =  %x30 / %x31-39 *( %x30-39 )
-           ; "0", or digits without a leading "0"
-]]>
-                </artwork>
-                <postamble>
-                    where &lt;json-pointer&gt; follows the production defined in
-                    <xref target="RFC6901">RFC 6901, Section 3</xref> ("Syntax").
-                </postamble>
-            </figure>
             <t>
+                The ABNF syntax of a Relative JSON Pointer is:
             </t>
+            <sourcecode type="abnf9110"><![CDATA[
+  relative-json-pointer = origin-specification json-pointer
```

Comment:
According to the only online ABNF validator I could find, it's not quite valid here.
The error is that `json-pointer` is not defined.
Removing it from that line seems fix the issue.

I'm not super familiar with ABNF to know if this is a legitimate issue or not.

Reply:
Okay. I'll have a look later.  I didn't actually check it.

Reply:
Looks like it was originally in the postamble on line 109 before the change.  Not sure if it was removed on purpose, but I think it should be in there. I can confirm offline.

Reply:
It says on line 103

```
; json-pointer from RFC 6901
```

Not sure if that's sufficient.

Reply:
Offline sources say that ABNF comment is sufficient to define `json-pointer` for this spec.  I'm happy with it.